projects/netgrif-components/src/lib/panel/panel-item/panel-item.component.html

```diff
@@ -0,0 +1,19 @@
+<span class='text-column-margin' fxLayoutAlign='start center'>
+    <mat-icon class='panel-icon panel-primary-icon' *ngIf='index === 0'>
```

Comment:
perhaps a better way would be to not provide the index, but a boolean that would directly specify weather the case icon should be displayed or not. That would make the component more readable in my opinion

Reply:
It is now refactored according to you.